### ISSUE TITLE
[FR] explain any disabled options 

### ISSUE THREAD

**Issue by ITwrx** [monero-project/monero-gui#1740]
i'm trying to send some xmr with monero gui 0.13.04 in arch linux and the send button is greyed out. i don't see anything wrong that would prevent this send action: balance is unlocked, wallet and daemon are synchronized, log shows no problems, etc. 

problems happen, but not telling the user why an option is disabled is infuriating and i don't think the experience we want people to have.

it would be much better, if anytime any option is disabled, the reason why, in plain english, is disclosed right beside the disabled option, just like a typical web form. if the gui can make the button grey and stop the user from submitting, it can show a tooltip or error.

the wallet is already not intuitive enough, imho, but disabling stuff without explanation or quick remedy is probably a deal breaker for many people. you have to be able to trust that the thing will work (or at least work with you) when you go to transfer funds.
thanks

i see #67 now but i don't have any red fields and i don't think a hint with a red field is good enough (without an explanation right there in the gui) anyways.

update: also with log level at 4 no problems are shown in the terminal.

**Comment by dEBRUYNE-1** [comment]
All right, please see this then:

https://monero.stackexchange.com/questions/10545/unable-to-send-amount-below-one-x-in-mac-os-and-windows-os-gui/

**Comment by ITwrx** [comment]
yes, that was the problem. thanks for your help with my specific issue. i'll leave this FR open until either the FR is implemented or monero devs decide to close it.

in this instance, if monero devs wanted to keep with the "highlighting of form fields" plan, then the amount field should have been red (and i would add a message shown by that field, as well)

**Comment by dEBRUYNE-1** [comment]
@ITwrx - I think the issue with the `Amount` field is covered by #1749 & #1750.

**Comment by ITwrx** [comment]
@dEBRUYNE-1 yeah, that works too. thanks for the info.

**Comment by dEBRUYNE-1** [comment]
All right. Going to mark this as resolved then.

**Comment by dEBRUYNE-1** [comment]
+resolved
